src/components/search/advancedSearch/popover/Popover.js

```diff
@@ -52,11 +57,21 @@ const Popover = ({ className, simpleSearchRef }) => {
         setShowInfoTooltip(false);
       }
     }
-    document.addEventListener("click", handleClickOutside);
+    document.addEventListener("mouseup", handleClickOutside);
     return () => {
-      document.removeEventListener("click", handleClickOutside);
+      document.removeEventListener("mouseup", handleClickOutside);
+    };
+  }, [popoverRef, showPopover, mouseDownPosition]);
+
+  useEffect(() => {
+    const handleMouseDown = (event) => {
+      setMouseDownPosition(event.pageY);
     };
-  }, [popoverRef, showPopover]);
+    window.addEventListener("mousedown", handleMouseDown);
```

Comment:
Skal det ikke være document i stedet for window?

Window rammer også scrollbar og selve browseren, mener jeg

Reply:
Kan godt være at det er lige meget. Jeg tror bare jeg ville forvente at begge var document eller begge var window